core/src/main/java/io/kestra/core/lock/Lock.java

```diff
@@ -0,0 +1,24 @@
+package io.kestra.core.lock;
+
+import io.kestra.core.utils.IdUtils;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Lock {
+    private String category;
+    private String id;
+    private String owner;
+    private LocalDateTime date;
```

Comment:
`created` or `createdAt` is more explicit than just `date`. Personally, I would use an `Instant` for such use case.